atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -160,7 +160,10 @@ def get_platforms(self):
     def get_current_builds(self, osbs):
         field_selector = ','.join(['status!={status}'.format(status=status.capitalize())
                                    for status in BUILD_FINISHED_STATES])
-        return len(osbs.list_builds(field_selector=field_selector))
+        try:
+            return len(osbs.list_builds(field_selector=field_selector))
+        except Exception:
+            return -1
```

Comment:
We must at least log the exception.

This is using an API method, so it's guaranteed to raise an OsbsException of some sort. Can we catch OsbsException instead of Exception?